.github/ISSUE_TEMPLATE/5_performance_speed.md

```diff
@@ -16,35 +16,35 @@ assignees: ''
       * https://api.flutter.dev/
       * https://stackoverflow.com/questions/tagged/flutter?sort=frequent
 
+     Issues directly asking for support will be closed.
+
      If you have found a performance problem, then fill out the template below.
      Please read our guide to filing a bug first: https://flutter.dev/docs/resources/bug-reports
 -->
 
 ## Details
 
 <!--
-1.  Please tell us exactly how to reproduce the problem you are running into.
-
-2.  Please attach a small application (ideally just one main.dart file) that
-     reproduces the problem. You could use https://gist.github.com/ for this.
```

Comment:
we seem to have lost #2 here?

Reply:
oh i see you moved it below. I really think this is the most important point though. Also the next paragraph says "this app" which makes no sense if we move #2 down.

Reply:
moved related sections down below